crates/ffmpeg/src/io/output.rs

```diff
@@ -281,3 +281,206 @@ impl Output<()> {
         })
     }
 }
+
+#[cfg(test)]
+#[cfg_attr(all(test, coverage_nightly), coverage(off))]
+mod tests {
+    use std::fs::File;
+    use std::io::Cursor;
+    use std::path::Path;
+
+    use crate::consts::DEFAULT_BUFFER_SIZE;
+    use crate::dict::Dictionary;
+    use crate::error::FfmpegError;
+    use crate::io::output::{AVCodec, AVOutputFormat};
+    use crate::io::{Output, OutputOptions};
+
+    #[test]
+    fn test_output_options_default() {
+        let options = OutputOptions::default();
+
+        assert_eq!(options.buffer_size, DEFAULT_BUFFER_SIZE);
+        assert!(options.format_name.is_none());
+        assert!(options.format_mime_type.is_none());
+        assert!(options.format_ffi.is_null());
+    }
+
+    #[test]
+    fn test_output_options_new() {
+        let options = OutputOptions::new();
+
+        assert_eq!(options.buffer_size, DEFAULT_BUFFER_SIZE);
+        assert!(options.format_name.is_none());
+        assert!(options.format_mime_type.is_none());
+        assert!(options.format_ffi.is_null());
+    }
+
+    #[test]
+    fn test_output_options_custom_values() {
+        let options = OutputOptions::default()
+            .buffer_size(4096)
+            .format_name("mp4")
+            .format_mime_type("video/mp4");
+
+        assert_eq!(options.buffer_size, 4096);
+        assert_eq!(options.format_name, Some("mp4"));
+        assert_eq!(options.format_mime_type, Some("video/mp4"));
+    }
+
+    #[test]
+    fn test_output_options_get_format_ffi() {
+        let options = OutputOptions::default().format_name("mp4");
+        let format_ffi = options.get_format_ffi();
+        assert!(format_ffi.is_ok());
+        assert!(!format_ffi.unwrap().is_null());
+    }
+
+    #[test]
+    fn test_output_options_get_format_ffi_null() {
+        let dummy_format_ffi: *const AVOutputFormat = 0x1234 as *const AVOutputFormat;
```

Comment:
Can we not get a real format from somewhere else instead of casting an integer to a pointer?